python/doc/examples/data_analysis/sample_analysis/plot_draw_survival.py

```diff
@@ -98,7 +98,7 @@
 p1 + p2
 
 # %%
-# The `drawCDF` and `drawSurvivalFunction` methods allows one to draw the functions :math:`F` and :math:`S`.
+# The `drawCDF` and `drawSurvivalFunction` methods allow one to draw the functions :math:`F` and :math:`S`.
```

Comment:
The sentence:

The `computeCDF` and `computeSurvivalFunction` computes the CDF :math:`F` and survival :math:`S` of a distribution.

can be replaced with:

The `computeCDF` and `computeSurvivalFunction` compute the CDF :math:`F` and survival :math:`S` of a distribution.

because there is, similarily, no "s" in the "compute" verb.